apps/web/src/components/Publication/FullPublication.tsx

```diff
@@ -36,10 +44,39 @@ const FullPublication: FC<Props> = ({ publication }) => {
     : publication?.stats?.totalAmountOfCollects;
   const showStats = mirrorCount > 0 || reactionCount > 0 || collectCount > 0;
 
+  const isGatedThread = !publication?.isGated && !commentOn?.isGated && !mainPost?.isGated;
+  // [...getURLs(mainPost?.metadata.content ?? ''), ...getURLs(commentOn?.metadata.content ?? '')].length ===
+  // 0;
+
+  const scrollToThread = () => {
+    if ((!mainPost && !commentOn) || !threadRef.current) {
+      return;
+    }
+    if (isGatedThread) {
```

Comment:
```suggestion
    }

    if (isGatedThread) {
```

added space between two statements 🙇🏼